src/Microsoft.Graph.Auth/Microsoft.Graph.Auth.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Microsoft Graph Auth Library implements auth functionality used by Microsoft Graph Client Libraries.</Description>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <AssemblyTitle>Microsoft Graph Auth Library</AssemblyTitle>
+    <VersionPrefix>0.1.0</VersionPrefix>
+    <FileVersion>0.1.0</FileVersion>
+    <AssemblyVersion>0.1.0</AssemblyVersion>
+    <Authors>Microsoft</Authors>
+    <TargetFrameworks>netstandard1.3;net45;</TargetFrameworks>
+    <AssemblyName>Microsoft.Graph.Auth</AssemblyName>
+    <PackageId>Microsoft.Graph.Auth</PackageId>
+    <PackageTags>Microsoft Office365;Graph;GraphServiceClient;Outlook;OneDrive;AzureAD;GraphAPI;Productivity;SharePoint;Intune;SDK</PackageTags>
+    <PackageReleaseNotes>
+    </PackageReleaseNotes>
+    <PackageProjectUrl>https://developer.microsoft.com/graph</PackageProjectUrl>
+    <PackageLicenseUrl>http://aka.ms/devservicesagreement</PackageLicenseUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
```

Comment:
Remove GenerateAssemblyTitleAttribute, GenerateAssemblyDescriptionAttribute, through GenerateAssemblyFileVersionAttribute, build, and compare the dll attributes. Currently we produce:

![image](https://user-images.githubusercontent.com/8527305/54161896-ee519400-4410-11e9-9fb5-9ada3421d27f.png)